### PR TITLE
docs: remove blob related specs

### PR DESCRIPTION
**Pull request by ghost**
This PR targets the release branch `release-1.1` only. 

It removes the blob sign/verify related contents based on the release plan.

### CODE REVIEW

File: specs/signing-and-verification-workflow.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ The user wants to pull an OCI artifact only if they are signed by a trusted publ
     1. **Filter signature artifact manifest:**
         1. Filter out the unsupported signature formats by comparing the signature envelope format type (`[descriptors].descriptor.mediaType`) in the signature manifest, with the supported formats defined in [signature specification](./signature-specification.md#storage).
         1. Depending upon the trust-store and trust-policy configuration, further filter out signature manifests.
-            1. Using the `scopes` configured in trust policies, get the applicable trust policy.
+            1. Using the `registryScopes` configured in trust policies, get the applicable trust policy.
```
Comment (shizhMSFT) [inline specs/signing-and-verification-workflow.md:53]: Which PR introduce the change of `scopes` to `registryScopes`?

File: specs/trust-store-trust-policy.md [hunk 2]
```diff
@@ -85,6 +83,7 @@ Users who consume signed artifacts from OCI registries, or signed arbitrary blob
   - **`trustStores`**(*array of string*): This REQUIRED property specifies a set of one or more named trust stores, each of which contain the trusted roots against which signatures are verified. Each named trust store uses the format `{trust-store-type}:{named-store}`. Currently supported values for `trust-store-type` are `ca`, `signingAuthority` and `tsa`. To enable timestamp verification, type `tsa` MUST be configured.
   - **`trustedIdentities`**(*array of strings*): This REQUIRED property specifies a set of identities that the user trusts. For X.509 PKI, it supports list of elements/attributes of the signing certificate's subject. For more information, see [identities constraints](#trusted-identities-constraints) section. A value `*` is supported if user trusts any identity (signing certificate) issued by the CA(s) in `trustStore`. This field only contains trusted identities issued by CA(s) or Signing Authorities. 
   
+  
 #### Blob Trust Policy
```
Comment (shizhMSFT) [inline specs/trust-store-trust-policy.md]: Blob trust policy is not removed and is duplicated.

File: specs/signing-and-verification-workflow.md (hunk 1 above)
Comment (shizhMSFT) [reply specs/signing-and-verification-workflow.md:53]: If it is a fix, please also create a PR to the `main` branch.

**Review by shizhMSFT — COMMENTED** [review]
I also took a look at https://github.com/notaryproject/specifications/compare/v1.0.0...Two-Hearts:specifications:release

File: specs/signing-and-verification-workflow.md (hunk 1 above)
Comment (ghost) [reply specs/signing-and-verification-workflow.md:53]: Yeah, we don't have a term called `scopes` in our trust policy. We should fix it in the main branch as well.

File: specs/trust-store-trust-policy.md (hunk 2 above)
Comment (ghost) [reply specs/trust-store-trust-policy.md]: Nice catch!

File: specs/signing-and-verification-workflow.md (hunk 1 above)
Comment (ghost) [reply specs/signing-and-verification-workflow.md:53]: Created a quick fix PR to main branch: https://github.com/notaryproject/specifications/pull/312.

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Review by rgnote — APPROVED** [review]
LGTM

**Review by yizha1 — APPROVED** [review]
LGTM

**Pull request merged**